recipe/meta.yaml

```
@@ -10,26 +10,39 @@ source:
   sha256: 3bf76a66361242b66adbecf332071b887143bf82f2ab7a3aabee5e54d7cdad09
 
 build:
-  noarch: python
-  number: 0
-  script: python -m pip install . -vv
-
-requirements:
-  host:
-    - python >=3.6
-    - pip
-  run:
-    - python >=3.6
-    - rdkit
-
-test:
-  imports:
-    - cmiles
-  requires:
-    - pip
-  commands:
-    - pip check
-    - python -c "import cmiles; print(cmiles.__version__)"
+  number: 1
+
+outputs:
+  - name: cmiles-base
+    build:
+      script: python -m pip install --no-deps .
```

I think the CF idiom uses a Jinja variable to make sure we get the `host` Python in case PATH mangling is done my dependencies. Probably a good idea anyway. Also a couple of verbose flags:

```suggestion
      script: {{ PYTHON }} -m pip install --no-deps . -vv
```

---

I had a few bad iterations with where to put this and fell back to more or less copying this: https://github.com/conda-forge/typing_extensions-feedstock/blob/master/recipe/meta.yaml

Switching it back now

---

That didn't work, and may not have been working before; asked in chat and for some reason, this block just doesn't work like that inside of `outputs:` (until somebody patches `conda-build`) so switching back to a script file